.github/workflows/analyze_and_test.yml

```
@@ -0,0 +1,51 @@
+name: Pull Request - Code Analysis and Unit Tests
+on:
+  pull_request:
+    branches: [ dev ]
+env:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+jobs:
+  build_test:
+    runs-on: ubuntu-latest
+    name: Build & Test
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core 6.0
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.*
+    - name: Submodule Init
+      run: |
+        git submodule init && git submodule update
+    - name: Static Code Analysis
+      run: |
+        npm install pnpm -g &&
+        dotnet build src/Utilities/Lombiq.Npm.Targets/Lombiq.Npm.Targets.csproj
+        dotnet build Lombiq.OSOCE.sln --no-incremental -warnaserror /p:TreatWarningsAsErrors=true /p:RunAnalyzersDuringBuild=true -nologo -consoleLoggerParameters:NoSummary -verbosity:quiet
+    - name: Build
+      run: |
+        dotnet build --configuration Release
+    - name: Set up SQL Server via Docker
+      run: |
+        mkdir -p ~/.local/docker/mssql/data
+        chown $USER:docker ~/.local/docker/ -R
+        chmod g+w ~/.local/docker/ -R
+        docker pull mcr.microsoft.com/mssql/server
+        docker volume create --driver local -o o=bind -o type=none -o device=~/.local/docker/mssql/data mssql-data
+        docker run --name sql2019 -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=yourStrong(!)Password' -v mssql-data:/data -p 1433:1433 -d 'mcr.microsoft.com/mssql/server:2019-latest'
+        docker exec -u 0 sql2019 bash -c 'chown mssql:root /data -R'
+        echo LOMBIQ_SHARED_TEST_CONFIGURATION="$PWD/.github/TestConfig.json" >> $GITHUB_ENV
+    - name: Set up Azurite via Docker
+      run: |
+        docker pull mcr.microsoft.com/azure-storage/azurite &&
+        docker run --name azurite -d -p 10000:10000 mcr.microsoft.com/azure-storage/azurite azurite-blob --blobHost 0.0.0.0 --blobPort 10000
+    - name: Unit Tests
```

Since these are also UI tests:

```suggestion
    - name: Tests
```